src/hardware/gus.cpp

```diff
@@ -1563,16 +1563,30 @@ void Gus::WriteToRegister()
 			StartDmaTransfers();
 		}
 		return;
-	case 0x4c: // Runtime control
+	case 0x4c: // Reset register
 	{
-		const auto state = (register_data >> 8) & 7;
-		if (state == 0) {
+		// The reset register bits control runtime state such as
+		// stoping, starting, and if the DAC or IRQs will be used.
+
+		const auto reset_register = static_cast<uint8_t>(register_data >> 8);
+
+		// From section 2.6.1.9, page 16, of the UltraSound Software
+		// Development Kit (SDK) Version 2.22
+
+		// Bit 0 - GF1 Master Reset. 0 = reset, 1 = run. As long as this
+		//         is a 0, it will be held in a reset state.
+		if (bit::cleared(reset_register, bit::literals::b0)) {
 			StopPlayback();
-		} else if (state == 1) {
+		} else {
 			PrepareForPlayback();
-		} else if (active_voices) {
-			BeginPlayback();
 		}
+
+		// Bit 1 - Enable DAC output. DAC's will not run unless its set.
+		dac_enabled = bit::is(reset_register, bit::literals::b1);
+
+		// Bit 2 - Master IRQ enable. This bit MUST be set to get ANY of
+		//         the GF1-generated IRQs (wavetable, volume, etc).
+		irq_enabled = bit::is(reset_register, bit::literals::b2);
```

Comment:
Very nice @interloper98! Bonus points for including the GUS SDK reference 🎉 

For _even more bonus points_ 😆, consider using our quite nice and self-documenting `bit_view` template for the Reset Register. Check out `vga.h` for usage examples. Entirely optional, though—this can be merged as-is if you don't feel like spending more time on this.

Reply:
I'm in it for the bonus points, so following those notes :-) Thanks for the tips @johnnovak

Reply:
> Looking at the code now, the reset register is just the tip of the iceberg that can benefit from it compared to lots of larger registers in the GUS.

Thanks for doing this, and yeah, that's right; `bit_view` is great for making all that low-level register-banging code self-documenting.

If you wanna convert everything in GUS to `bit_view` in a follow-up PR, that would be a most welcome change 😉 😉

Reply:
Thanks; I might keep poking around! (probably far too much to bite off; I got lucky with this one and it worked out OK!)